### PR TITLE
FY 23 Q1 Roadmap Updates and September Release Notes

### PR DESCRIPTION
**Pull request by amanifarooque**
## PR Checklist
- [x] Test Federalist Preview link for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
- [x] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [x] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

**Comment by amanifarooque** [comment]
Release Notes: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/roadmap-updates/about/updates/releases/september-2022.html

Roadmap: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/roadmap-updates/about/updates/roadmap/

**Review by MyNameIsMissing — APPROVED** [review]
LGTM

File: about/updates/roadmap/index.md [hunk 1]
```diff
@@ -17,54 +17,53 @@ sidenav: about_us
 
 At Search.gov we aim to provide an excellent public search experience, a self-service search administration experience, and a rock-solid secure and compliant infrastructure to support it all, while keeping costs down across the government.
 
-We know it’s helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms. 
+We know it's helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms.
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:20]: @amanifarooque will this straight apostrophe display differently than the curly/angled apostrophe later in the sentence?

File: about/updates/roadmap/index.md [hunk 2]
```diff
@@ -17,54 +17,53 @@ sidenav: about_us
...
+* **Javascript Indexing - Production Release**
+  * We will be getting our Javascript indexing feature tested and ready for production.
 * **Search Engine Results Page (SERP) Redesign - Investigations**
   * We will begin scoping the work required to upgrade our frontend technology in support of this redesign.
-* **Continuity of Operations Plan Improvements** 
-  * We are taking steps to bolster our system's resiliance and reliability.
-* **Security and Logging Improvements**
-  * We are adding additional defenses to protect our system from attack, and ways to better visualize incoming traffic. 
+* **Version Upgrades**
+  * We will be upgrading our ElasticSearch, Ruby, and Redis versions to keep them up-to-date and compliant.
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:36]: Also Rails now?

File: about/updates/roadmap/index.md [hunk 3]
```diff
@@ -17,54 +17,53 @@ sidenav: about_us
...
 * **Search Engine Results Page (SERP) Redesign - Investigations**
   * We will begin scoping the work required to upgrade our frontend technology in support of this redesign.
-* **Continuity of Operations Plan Improvements** 
-  * We are taking steps to bolster our system's resiliance and reliability.
-* **Security and Logging Improvements**
-  * We are adding additional defenses to protect our system from attack, and ways to better visualize incoming traffic. 
+* **Version Upgrades**
+  * We will be upgrading our ElasticSearch, Ruby, and Redis versions to keep them up-to-date and compliant. 
+* **RSS Feed Indexing**
+  * In addition to sitemap indexing, we will add support for automated indexing of RSS feeds.
```
Comment (dawnpm) [inline about/updates/roadmap/index.md:38]: `...RSS feeds into our primary web index.` ?

**Review by dawnpm — CHANGES_REQUESTED** [review]
ooo I love this roadmap! just a few questions, see comments. thanks!!!

**Comment by amanifarooque** [comment]
Style Guide updates to make: 

we recommend:
Search.gov Roadmap Fiscal Year 2022 Quarter 4
and
FY 2022 Q4: July-September 2022

File: about/updates/roadmap/index.md (hunk 1 above)
Comment (amanifarooque) [reply about/updates/roadmap/index.md:20]: Trying to make these consistent, made a couple more replacements.

File: about/updates/roadmap/index.md (hunk 2 above)
Comment (amanifarooque) [reply about/updates/roadmap/index.md:36]: Updated

File: about/updates/roadmap/index.md (hunk 3 above)
Comment (amanifarooque) [reply about/updates/roadmap/index.md:38]: Updated

File: about/updates/roadmap/index.md (hunk 1 above)
Comment (dawnpm) [reply about/updates/roadmap/index.md]: ok, thanks

**Review by dawnpm — APPROVED** [review]
Added mdash where needed, ready to post. thanks!

**Comment by amanifarooque** [comment]
Thank you Dawn!

**Pull request merged**